pkgs/by-name/tu/tuxguitar/package.nix

```
@@ -22,8 +24,13 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-FD1+7jV69E9AfTczjD6DOGD+pPlscg4o8A9ADBUM9B4=";
   };
 
+  buildInputs = [
+    which
+  ];
+
   nativeBuildInputs = [
     makeWrapper
+    wrapGAppsHook
```

```suggestion
    wrapGAppsHook3
```
https://github.com/NixOS/nixpkgs/pull/307077